wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/AccountViews/AccountViewBuilder.swift

```diff
@@ -26,6 +26,8 @@ struct AccountViewBuilder {
 
     func build() -> BaseAccountView {
 
+        // TODO: [WPB-7307] availability status must be shown on the avatar image (right-bottom)
```

Comment:
this is remove folders ticket, is it still relevant?

Reply:
I don't think so, removed it.